core/tests/webdriverio_utils/ReleaseCoordinatorPage.js

```diff
@@ -123,6 +123,58 @@ var ReleaseCoordinatorPage = function() {
     return null;
   };
 
+  // Remove this method after the cd_admin_dashboard_new_ui feature flag
```

Comment:
Create an issue for this and provide the link here. Prefix the comment with `TODO:`.

Reply:
Done!.